.github/workflows/coverage.yml

```diff
@@ -5,12 +5,15 @@ on:
   pull_request:
     branches:
       - main
-  push:
-    branches:
-      - main
```

Comment:
Do we need coverage to run on `main` to keep the base coverage up to date?
Or is coverage only stored by commit hash?
(If it is, then the branch doesn't matter.)

Reply:
I left it this way, so we can later add a percentage and status badge for coverage on the main branch, aside from that it's not needed.